src/rust/engine/fs/fs_util/src/main.rs

```diff
@@ -155,6 +155,12 @@ to this directory.",
                   .default_value("binary")
                   .possible_values(&["binary", "recursive-file-list", "recursive-file-list-with-digests", "text"]),
               )
+                .arg(
+                  Arg::with_name("child-dir")
+                      .long("child-dir")
+                      .takes_value(true)
+                      .help("Child directory to navigate to in the proto before operating.")
```

Comment:
Could expand the help a bit to indicate that this is the relative path of a directory inside the digest?